server/middleware/populateSelectedEstablishment.ts

```diff
@@ -1,16 +1,16 @@
 import type { RequestHandler } from 'express'
-import SupportedPrisonsService from '../services/supportedPrisonsService'
 import asyncMiddleware from './asyncMiddleware'
+import { Services } from '../services'
 
-export default function populateSelectedEstablishment(
-  supportedPrisonsService: SupportedPrisonsService,
-): RequestHandler {
+export default function populateSelectedEstablishment({ supportedPrisonsService }: Services): RequestHandler {
   return asyncMiddleware(async (req, res, next) => {
     if (req.session.selectedEstablishment === undefined && req.path !== '/change-establishment') {
       const supportedPrisons = await supportedPrisonsService.getSupportedPrisons(res.locals.user.username)
 
       const { activeCaseLoadId } = res.locals.user
 
+      // TODO need to handle BOTH selectedEstablishment and activeCaseLoadId not being set
```

Comment:
Not actually an issue whilst we still have the 'Change establishment' page in place. Will be resolved when that gets removed.